docs/build-tools/sbt.md

```diff
@@ -225,7 +225,7 @@ classifier.
 
 ### Export main class from sbt
 
-If you want bloop to export `mainClass` from your build definition, define either of the following
+If you want bloop to export `mainClass` from your build definition, define the following
 settings in your `build.sbt`:
```

Comment:
Since you're fixing this, do you mind also changing the next line as well:

```suggestion
setting in your `build.sbt`:
```

Reply:
good catch. fixed=)